### ISSUE TITLE
OnDidLayoutChange may be missing when initializing editor

### ISSUE THREAD

**Issue by whyour** [suren-atoyan/monaco-react#591]
**Describe the bug**
I need to listen to the onDidLayoutChange event, but I won't get the editor instance until after the onMount event, maybe the first few onDidLayoutChanges have been executed before onMount. 

**Expected behavior**
Add onDidLayoutChange property to Editor.

**Comment by suren-atoyan** [comment]
@whyour there is also `beforeMount` prop

**Comment by xueerli** [comment]
@suren-atoyan 
The beforeMount event doesn't have an editor instance.
Another way is to use `monaco.editor.onDidCreateEditor` to get the editor as soon as possible and listen to the onDidLayoutChange event.
```
monaco.editor.onDidCreateEditor(editor => {
      const off = editor.onDidLayoutChange(()=> {
      });
      editor.onDidDispose(() => {
        off?.dispose();
      });
})
```

**Comment by suren-atoyan** [comment]
Right, sorry, I was thinking you are looking for a `monaco` instance.

In that case, `monaco.editor.onDidCreateEditor` should work, isn't it?

**Comment by xueerli** [comment]
Yes, it works.
But it would be nice to add a prop so that something could be done with the context inside the component.
The `monaco.editor.onDidCreateEditor` can only be listened to globally once, and there is no more information here except for the editor instance.
